include/miopen/miopen.h

```diff
@@ -2690,6 +2699,27 @@ miopenDestroyActivationDescriptor(miopenActivationDescriptor_t activDesc);
 /** @} */
 // CLOSEOUT ACTIVATION DOXYGEN GROUP
 
+// Gemm APIs
+/** @addtogroup gemm
+ *
+ *  @{
+ */
+/*! @brief Creates the Gemm descriptor object
+ *
+ * @param gemmDesc Pointer to an gemm tensor descriptor type
+ * @return          miopenStatus_t
+ */
+MIOPEN_EXPORT miopenStatus_t miopenCreateGemmDescriptor(miopenGemmDescriptor_t* gemmDesc);
+
+MIOPEN_EXPORT miopenStatus_t miopenInitGemmDescriptor(miopenGemmDescriptor_t gemmDesc,
+                                                      int m_,
+                                                      int n_,
+                                                      int k_,
+                                                      long long int ldA_,
+                                                      long long int ldB_,
+                                                      long long int ldC_,
+                                                      miopenDataType_t dataType_);
```

Comment:
For convolution we dont include the input sizes and types in the convolution descriptor. We should not include it in the gemm descriptor either.